.github/workflows/build-all-arduino.yml

```
@@ -0,0 +1,87 @@
+        tag_name: ${{ github.ref_name }}
+        name: Release ${{ github.ref_name }}
+        body: Automatic release of firmware version ${{ github.ref_name }}
+        files: artifacts/*.bin
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+    - name: ⏩ Skip existing release
+      if: steps.check-release.outputs.exists == 'true'
+      run: |
+        echo "Skipping build/release: Release already exists for ${{ github.ref_name }}"
+  
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)